pkgs/tools/typesetting/sile/default.nix

```diff
@@ -1,27 +1,29 @@
 { lib
 , stdenv
 , darwin
-, fetchurl
+, fetchurl, zstd
 , makeWrapper
 , pkg-config
-, poppler_utils
```

Comment:
I'm a bit confused about all of those formatting and reorder changes that enlarge the diff (for so no real reason so it seems). I also noticed your other reformat PR upstream:

https://github.com/sile-typesetter/sile/pull/2081

What is the reason behind all of this changes? Which expression should lead the formatting standard? BTW we have nowadays a consensus sort of regarding formatting, it is contained in the package [`nixfmt-rfc-style`](https://search.nixos.org/packages?channel=unstable&show=nixfmt-rfc-style&from=0&size=15&sort=relevance&type=packages&query=nixfmt).

Whatever decision you'll go with, I'd suggest to apply a `nixfmt` + consistent argument reordering, and then perform the changes related to the update itself - it will make the PR a bit easier to review. Another option is to discard any of the formatting changes, and minimize the diff as much as possible, which is also legitimate.

Reply:
The re-ordering was because both the package here and the upstream flake had different orders making them hard to diff, and *neither* order made any sense to me. Re-ordering them both grouping by type and then alphabetically brought some rhyme and reason to them. I'll look into `nixfmt`.

Reply:
I have added some comments to the dependency lists marking logical groups, then sorted them alphabetically in each group. I've also reformatted the whole thing with `nixfmt-rfc-style`, and then separately bumped the version. That should make the version bump related changes much easier to review.

Sorry about the delay, life just got a bit chaotic to put it mildly.